### PR TITLE
feat(vinnie): add shrimp to the list

### PR DESCRIPTION
**Pull request by Porthorian**
## Purpose

We need to add shrimp to the list this is topic of much need.

### CODE REVIEW

File: vinnie.md [hunk 1]
```diff
@@ -16,7 +16,8 @@
 16. Fried Rice
 17. BLT
 18. Sushi
-19. Fish 
-20. Pizza
-21. Stromboli
+19. Shrimp
+20. Fish 
+21. Pizza
```
Comment (zamerendino) [inline vinnie.md:21]: this list is supposed to be a source of TRUTH. All I see here is lies and deceit. We have not poured out life savings into this project for you to immediately make a mockery out of it. Pathetic.

**Review by zamerendino — CHANGES_REQUESTED** [review]

File: vinnie.md (hunk 1 above)
Comment (zamerendino) [reply vinnie.md:21]: Pizza at 21??? I will sue.

**Review by aanthonyrizzo — CHANGES_REQUESTED** [review]
I dont believe you

**Comment by aanthonyrizzo** [comment]
Great commit message format

**Comment by zamerendino** [comment]
move pizza up to 15 at least and we got a deal here

**Comment by ddymko** [comment]
I have _NEVER_ seen you eat shrimp or even order shrimp. Now, don't say "we never got it" because there are plenty of places we order from where we can get shrimp.

File: vinnie.md (hunk 1 above)
Comment (Porthorian) [reply vinnie.md:21]: Listen @zamerendino this took weeks to do and decide on this ordering. If you don't like it I'll take it up to the CTO. You signed a binding agreement to do thorough code reviews and this has not been a value add. Why are we paying you all these pennies?

**Comment by Porthorian** [comment]
> I have _NEVER_ seen you eat shrimp or even order shrimp. Now, don't say "we never got it" because there are plenty of places we order from where we can get shrimp.

@ddymko I never got it cause I don't trust it from any place I'd order takeout from haha

**Comment by zamerendino** [comment]
I refuse to let shrimp sit above pizza on your list.

**Comment by Porthorian** [comment]
@zamerendino had to merge this in urgent bug fix.

**Pull request merged**